### PR TITLE
[ci-visibility] Add explanation for test suite visibility

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
Add explanation for test suite level visibility. 

### Motivation
Test suite level visibility is a new feature that we want to explain to users. 

### Preview
[New test suite level visibility section](https://docs-staging.datadoghq.com/juan-fernandez/add-test-suite-level-visibility/continuous_integration/tests/#test-suite-level-visibility)

[Javascript compatibility](https://docs-staging.datadoghq.com/juan-fernandez/add-test-suite-level-visibility/continuous_integration/tests/javascript/?tab=onpremisesciproviderdatadogagent#test-suite-level-visibility-compatibility)

[.NET compatibility](https://docs-staging.datadoghq.com/juan-fernandez/add-test-suite-level-visibility/continuous_integration/tests/dotnet/?tab=onpremisesciproviderdatadogagent#test-suite-level-visibility-compatibility)

[Swift compatibility](https://docs-staging.datadoghq.com/juan-fernandez/add-test-suite-level-visibility/continuous_integration/tests/swift/?tab=swiftpackagemanager#test-suite-level-visibility-compatibility)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/_index.md
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/_index.md:99]: while this is not true for Swift, I'm not sure whether it makes sense to talk about that here.

File: content/en/continuous_integration/tests/_index.md
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/_index.md:123]: since when is test suite level visibility supported in Swift @nachoBonafonte ?

File: content/en/continuous_integration/tests/dotnet.md
Comment (tonyredondo) [inline content/en/continuous_integration/tests/dotnet.md:34]: 2.16.0

File: content/en/continuous_integration/tests/_index.md
Comment (tonyredondo) [reply content/en/continuous_integration/tests/_index.md]: 2.16.0

**Comment by cswatt** [comment]
created DOCS-4286 to track

**Review by jtappa — APPROVED** [review]

**Comment by jtappa** [comment]
Looks like there's some merge conflicts that need to be resolved before merge, did you also want to wait on review from @nachoBonafonte?

**Review by nachoBonafonte — APPROVED** [review]

**Comment by fermayo** [comment]
@jtappa conflicts resolved!

**Pull request merged**
